File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
a=[]
for i in range(3):
  n=int(input())
  a.append(n)
if(sum(a)!=180):
  print("Error")
else:
  if(a.count(60)==3):
    print("Equilateral")
  else:
    r=0
    for i in a:
      if(a.count(i)==2):
        print("Isosceles")
        r=1
        break
    if(r==0):
      print("Scalene")

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.io.*;
import java.util.*;
public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[] tri = new int[3];
		int total = 0;
		for(int i = 0; i < 3; i++) {
			tri[i]= Integer.parseInt(br.readLine());
			total+=tri[i];
		}
		if(total!=180) {
			System.out.println("Error");
		}else {
			boolean isit = false;
			for(int i = 0; i< 3; i++) {
				for(int j = i+1; j <3; j++) {
					if(tri[i]== tri[j]) {
						isit= true;
						break;
					}
				}
			}
			if(isit && tri[0]== 60) {
				System.out.println("Equilateral");
			}else if(!isit) {
				System.out.println("Scalene");
			}else {
				System.out.println("Isosceles");
			}
		}
		
	}
}
